internal/scorecard/alpha/scorecard.go

```diff
@@ -17,54 +17,74 @@ package alpha
 import (
 	"errors"
 	"fmt"
-	"io/ioutil"
-	"log"
 	"strings"
+	"time"
 
 	"github.com/operator-framework/operator-sdk/version"
-	"gopkg.in/yaml.v2"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/kubernetes"
 )
 
 type Options struct {
-	Config       Config
-	Selector     labels.Selector
-	List         bool
-	OutputFormat string
-	Client       kubernetes.Interface
+	Config          Config
+	Selector        labels.Selector
+	BundlePath      string
+	WaitTime        int
+	OutputFormat    string
+	Kubeconfig      string
+	Namespace       string
+	BundleConfigMap *v1.ConfigMap
+	ServiceAccount  string
+	Client          kubernetes.Interface
+	List            bool
+	Cleanup         bool
 }
 
 // RunTests executes the scorecard tests as configured
 func RunTests(o Options) error {
 	tests := selectTests(o.Selector, o.Config.Tests)
+	if len(tests) == 0 {
+		fmt.Println("no tests selected")
+		return nil
+	}
+
+	bundleData, err := getBundleData(o.BundlePath)
+	if err != nil {
+		fmt.Printf("error getting bundle data %s\n", err.Error())
+		return nil
+	}
+
+	// create a ConfigMap holding the bundle contents
+	o.BundleConfigMap, err = createConfigMap(o, bundleData)
+	if err != nil {
+		fmt.Printf("error creating ConfigMap %s\n", err.Error())
+		return nil
+	}
 
 	for i := 0; i < len(tests); i++ {
-		if err := runTest(tests[i]); err != nil {
+		var err error
+		tests[i].TestPod, err = runTest(o, tests[i])
+		if err != nil {
 			return fmt.Errorf("test %s failed %s", tests[i].Name, err.Error())
 		}
```

Comment:
If we're running in parallel, another approach would be to handle each pod in isolation by using goroutines and `sync.WaitGroup`. Something like:
```go
ctx, cancel := context.WithTimeout(context.Background(), o.WaitTime)
defer cancel()

var wg sync.WaitGroup{}
wg.Add(len(tests))

testResults := make(scorecardv1alpha1.TestResult, len(tests))
for idx, test := range tests {
    go func(i int, t Test) {
        defer wg.Done()
        result, err := runTest(ctx, o, test)
        if err != nil {
            result = convertErrorToResult(err)
        }
        testResults[i] = result
    }(idx, test)
}
wg.Wait()
return testResults
```

The beauty of this is that if we wanted to run in series, this would simplify to:
```go
ctx, cancel := context.WithTimeout(context.Background(), o.WaitTime)
defer cancel()

testResults := make(scorecardv1alpha1.TestResult, len(tests))
for i, t := range tests {
        result, err := runTest(ctx, o, test)
        if err != nil {
            result = convertErrorToResult(err)
        }
        testResults[i] = result
}
return testResults
```

In this example, `runTest` would create the pod, wait for just that pod to finish, grab the logs, parse it into a `TestResult` and then return it.

I almost think this is simple enough that we could go ahead and add parallel vs in series support in the config file and then do one or the other of the above.